app/models/link.rb

```diff
@@ -0,0 +1,14 @@
+class Link < ActiveRecord::Base
```

Comment:
For the future... :)  This link logic should be separated out from the `create!` logic in the applications_controller into a separate PR